Modules/Core/Common/test/itkXMLFileOutputWindowTest.cxx

```diff
@@ -46,6 +56,33 @@ itkXMLFileOutputWindowTest(int argc, char * argv[])
 
   logger->SetInstance(logger);
 
+  // If not input filename is provided, remove the contents in the existing file to avoid counting existing lines when
+  // contents are appended
+  if (argc == 1)
+  {
+    // In order to initialize the filename, some text needs to be written first
+    const char * regularText = "text";
+    logger->DisplayText(regularText);
+
+    // Get the filename
+    const char *      fileBaseName = logger->GetFileName();
+    const std::size_t size = 4096;
+    char              tmp[size];
+    char *            status = cwd(tmp, size);
+    if (!status)
```

Comment:
Also, although unlikely, if `getcwd` requires a larger buffer size, the error should be dealt with. Please feel free to improve this. Thanks.